lib/matplotlib/widgets.py

```diff
@@ -2711,10 +2768,14 @@ def onselect(eclick: MouseEvent, erelease: MouseEvent)
         - "move": Move the existing shape, default: no modifier.
         - "clear": Clear the current shape, default: "escape".
         - "square": Make the shape square, default: "shift".
-        - "center": Make the initial point the center of the shape,
-          default: "ctrl".
+        - "center": change the shape around its center, default: "ctrl".
+        - "data_coordinates": define if data or figure coordinates should be
```

Comment:
It seems weird to have this as part of the state that can change. I think it would be better as a property that is fixed once the selector is created.

Reply:
Yes, I agree that it doesn't feel very natural. I guess that I have done it that way because of the it follow a similar pattern as rotate/center and it is useful for testing or debugging purpose, but there are not really good reasons! :)
I am happy to have it as a property.

Reply:
Yes, I think we should have this fixed when the selector is created.